src/content/docs/configuration/v4_migration.md

```diff
@@ -3,132 +3,150 @@ id: v4_migration
+| `icons`                      | `AstroNvim/astroui`   | `icons`                     |
+| `lsp.capabilities`           | `AstroNvim/astrolsp`  | `capabilities`              |
+| `lsp.config.<lsp>`           | `AstroNvim/astrolsp`  | `config.<lsp>`              |
+| `lsp.flags`                  | `AstroNvim/astrolsp`  | `flags`                     |
+| `lsp.formatting`             | `AstroNvim/astrolsp`  | `formatting`                |
+| `lsp.mappings`               | `AstroNvim/astrolsp`  | `mappings`                  |
+| `lsp.on_attach`              | `AstroNvim/astrolsp`  | `on_attach`                 |
+| `lsp.servers`                | `AstroNvim/astrolsp`  | `servers`                   |
+| `lsp.setup_handlers`         | `AstroNvim/astrolsp`  | `handlers`                  |
+| `lsp.skip_setup`             | `AstroNvim/astrolsp`  | Set `handler.<lsp> = false` |
+| `mappings`                   | `AstroNvim/astrocore` | `mappings`                  |
+| `options`                    | `AstroNvim/astrocore` | `options`                   |
+| `text_icons`                 | `AstroNvim/astroui`   | `text_icons`                |
+
+v3 keys configured in new ways in v4:
+
+| `user/init.lua` table key | Starter Template Migration                                                                                                    |
+| ------------------------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| `lazy`                    | `lazy` is bootstrapped through `init.lua` and configured using `lua/lazy_setup.lua`                                           |
+| `plugins`                 | Plugins are in the directory `lua/plugins/<any_files>.lua`                                                                    |
+| `polish`                  | `polish` is in the file `lua/polish.lua`                                                                                      |
+| `updater`                 | Updating uses `lazy` update features and is configured in `init.lua` using `lazy` `branch` and `version` plugin spec options. |
+
+The following keys are introduced in v4 and have no equivalent in v3. This configuration was done through user configuration (for example in `polish.lua):
+
+| New key         | v4 plugin             | Description                                                                    |
+| --------------- | --------------------- | ------------------------------------------------------------------------------ |
+| `autocmds`      | `AstroNvim/astrocore` | Configure global auto commands                                                 |
+| `commands`      | `AstroNvim/astrocore` | Configure global commands                                                      |
+| `features`      | `AstroNvim/astrocore` | Configuration options for AstroNvim itself; was a `vim.g` variable in v3       |
```

Comment:
we should explain each of these `features` sub fields just because each one corresponded to a different `vim.g` variable in v3